tests/test_pipelines_common.py

```diff
@@ -9,4 +18,79 @@ class PipelineTesterMixin:
     equivalence of dict and tuple outputs, etc.
     """
 
-    pass
+    def test_save_load(self):
+        device = "cpu"
+        components = self.get_common_pipeline_components()
+        pipe = self.pipeline_class(**components)
+        pipe.to(device)
+        pipe.set_progress_bar_config(disable=None)
+
+        inputs = self.get_common_inputs(device)
+        output = pipe(**inputs)[0]
+
+        with tempfile.TemporaryDirectory() as tmpdir:
+            pipe.save_pretrained(tmpdir)
+            pipe_loaded = self.pipeline_class.from_pretrained(tmpdir)
+            pipe_loaded.to(device)
+            pipe_loaded.set_progress_bar_config(disable=None)
+
+        output_loaded = pipe_loaded(**self.get_common_inputs(device))[0]
+
+        max_diff = np.abs(output - output_loaded).max()
+        self.assertLessEqual(max_diff, 1e-5)
+
+    def test_tuple_output(self):
```

Comment:
Nice! That's very good for now ! In the future we could expand this test to also automatically make sure that not only the first tuple is the same but all the others now. 

For now this would e.g. apply to the stable diffusion pipeline.